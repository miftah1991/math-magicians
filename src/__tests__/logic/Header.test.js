import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import Header from '../../components/header';

it('renders correctly', () => {
  const tree = TestRenderer.create(<Router><Header /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
