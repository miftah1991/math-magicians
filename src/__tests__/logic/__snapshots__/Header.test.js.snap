// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="nav"
>
  <h1
    className="nav__title"
  >
    Math Magicians
  </h1>
  <ul
    className="nav__links"
  >
    <li>
      <a
        className="nav__link"
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="nav__link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className="nav__link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
